mlir/utils/jenkins/Jenkinsfile

```diff
@@ -316,31 +316,33 @@ void parameterSweep(String CONFIG, String codepath) {
 
 boolean shouldRunFromCodepath(String codepath) {
     // Run vanilla on public CI
-    if ((codepath == "vanilla") && (params.canXdlops == false)) {
+    if ((codepath == "vanilla") && !params.canXdlops) {
         return true
     }
     // Run mfma on private CI
     if ((codepath == "mfma") && params.canXdlops) {
         return true
     }
     // Run navi21 on private nightly or weekly CI if it is not disabled
-    if (params.canXdlops && (params.disableNavi21 == false) && (codepath == "navi21") &&
+    if (params.canXdlops && !params.disableNavi21 && (codepath == "navi21") &&
         (params.nightly || params.weekly)) {
         return true
     }
     // Run navi3x on private CI if it is not disabled
-    if (params.canXdlops && (params.disableNavi3x == false) && (codepath == "navi3x")) {
+    if (params.canXdlops && !params.disableNavi3x && (codepath == "navi3x")) {
         return true
     }
     return false
 }
 
 boolean shouldRunFromChip(String chip) {
     switch (chip) {
-        case "gfx906":
```

Comment:
Summary:  default is vanilla, gfx90a is allowed for either mfma or vanilla (because that's all we have on the public side, where vanilla is run), and gfx906 isn't mentioned at all.